.golangci.yaml

```
@@ -19,6 +20,13 @@ linters:
     - usestdlibvars
 
 linters-settings:
+  importas:
+    alias:
+      # prevent conflicts with std packages
+      - pkg:  github.com/go-sprout/sprout/registry/time
+        alias: rtime
+      - pkg:  github.com/go-sprout/sprout/registry/strings
+        alias: rstrings
```

I see your issue #103, and you already add it here so I will link the issue, and to keep the same import logic across all registries, we can use the regex synthax to always have `r{package}` across the project 🥇 

```suggestion
      # prevent conflicts with std packages
      - pkg:  github.com/go-sprout/sprout/registry/(\w+)
        alias: r$1
```

---

I assumed you wanted me to fix the import aliases in this PR.

I had no idea the importas could do that